packages/basis/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@pixi/basis",
+  "version": "5.3.2",
+  "description": "Loader for the *.basis supercompressed texture file format. This package also ships with the transcoder!",
+  "keywords": [
+    "basisu",
+    "pixi",
+    "compressed-textures",
+    "loaders"
+  ],
+  "author": "Shukant Pal <shukantpal@outlook.com>",
+  "homepage": "https://github.com/pixijs/pixi.js#readme",
+  "license": "MIT",
+  "main": "lib/basis.js",
+  "module": "lib/basis.es.js",
+  "bundle": "dist/basis.js",
+  "directories": {
+    "lib": "lib",
+    "test": "test"
+  },
+  "files": [
+    "dist",
+    "lib",
+    "assets"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/pixijs/pixi.js.git"
+  },
+  "scripts": {
+    "test": "echo \"Error: run tests from root\" && exit 1"
+  },
+  "bugs": {
+    "url": "https://github.com/pixijs/pixi.js/issues"
+  },
+  "dependencies": {
+    "@pixi/core": "5.3.2",
+    "@pixi/constants": "5.3.2",
+    "@pixi/runner": "5.3.2",
+    "@pixi/loaders": "5.3.2",
+    "resource-loader": "3.0.1"
```

Comment:
Do not include resource-loader here. It's not used as a dependency and everything comes from @pixi/loaders.